flang/test/Transforms/debug-module-2.f90

```diff
@@ -0,0 +1,39 @@
+! RUN: %flang_fc1 -emit-llvm -debug-info-kind=standalone %s -o - | FileCheck %s
+! RUN: %flang_fc1 -emit-llvm -debug-info-kind=line-tables-only %s -o - | FileCheck --check-prefix=LINEONLY %s
```

Comment:
LLVM IR tests should go to the Integration directory.

Reply:
I have moved the testcase to Integration folder. There may be other IR testcase that I added and I will move them in separate PR.